tests/lib/KeyStore.spec.js

```diff
@@ -157,7 +166,8 @@ describe('KeyStore', () => {
         const key1 = await KeyStore.instance._get(Dummy.keyInfos[0].id);
         expect(key1).toEqual(Dummy.storedKeyRecords()[0]);
 
-        // TODO: Expect Accounts DB to not be deleted
+        const accountsDb = await AccountStore.instance.connect();
+        expect(accountsDb).toBe(null);
```

Comment:
Sure it should not be not.toBe? Can't take a deeper look right now.

Reply:
The comment was actually wrong :D
That test really does the migration, and the test is mainly if it also removes the flag-cookies.

I added the `.not.toBe(null)` in the Iframe tests, where only the cookie should be set.